src/openboardview/FileFormats/BRDFileBase.h

```diff
@@ -64,6 +64,7 @@ struct BRDPin {
 	double radius    = 0.5f;
 	const char *snum = nullptr;
 	const char *name = nullptr;
+	const char *comment = nullptr;
```

Comment:
Probably better to declare as an `std::string` already to avoid some memory management issues.